app/src/main/java/org/wikipedia/navtab/MenuNavTabDialog.kt

```diff
@@ -98,18 +99,31 @@ class MenuNavTabDialog : ExtendedBottomSheetDialogFragment() {
 
     private fun updateState() {
         if (AccountUtil.isLoggedIn) {
-            binding.mainDrawerAccountAvatar.setImageResource(R.drawable.ic_baseline_person_24)
-            ImageViewCompat.setImageTintList(binding.mainDrawerAccountAvatar, getThemedColorStateList(requireContext(), R.attr.secondary_color))
-            binding.mainDrawerAccountName.text = AccountUtil.userName
-            binding.mainDrawerAccountName.visibility = View.VISIBLE
-            binding.mainDrawerLoginButton.visibility = View.GONE
+            if (AccountUtil.isTemporaryAccount) {
+                binding.mainDrawerAccountAvatar.setImageResource(R.drawable.ic_login_24px)
+                ImageViewCompat.setImageTintList(binding.mainDrawerAccountAvatar, getThemedColorStateList(requireContext(), R.attr.progressive_color))
+                binding.tempAccountName.text = AccountUtil.userName
```

Comment:
Notice the `main_drawer_temp_account_container` is clickable (with the ripple animation) but does not have an action for this menu.

Reply:
done!